cypress/integration/azureQueryBuilder.spec.js

```diff
@@ -0,0 +1,249 @@
+describe('test query builder on Azure', () => {
```

Comment:
This test is basically the same as the queryBuilder test, but testing against the Azure version of the same dataset. I haven't yet figured out how to share the actual tests. Any ideas?